### PR TITLE
Adding RadGEEToolbox recipe

### PR DESCRIPTION
**Pull request by radwinskis**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by radwinskis** [comment]
@conda-forge/help-python the recipe has passed all checks and is ready for review. Thanks!

**Comment by radwinskis** [comment]
As it has been almost a month: @conda-forge/help-python the recipe has passed all checks and is ready for review. Thanks!

**Comment by radwinskis** [comment]
Hello @conda-forge/help-python and @conda-forge/staged-recipes,

I'm following up on my pull request submitted 3+ months ago and ready for review. I acknowledge your busy schedules but would be grateful for any updates or feedback.

Thank you

**Comment by radwinskis** [comment]
@conda-forge-admin, please ping team

File: recipes/radgeetoolbox/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - earthengine-api
+    - python >=3.6
+
+test:
+  imports:
+    - RadGEEToolbox
```
Comment (ocefpaf) [inline recipes/radgeetoolbox/meta.yaml:27]:
```suggestion
test:
  imports:
    - RadGEEToolbox
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/radgeetoolbox/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 880ef49e211db36f8197cbff9da194fc61f496b5231852322da4394520897e6f
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation"
```
Comment (ocefpaf) [inline recipes/radgeetoolbox/meta.yaml:15]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
  sha256: 880ef49e211db36f8197cbff9da194fc61f496b5231852322da4394520897e6f

build:
  number: 0
  noarch: python
  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

File: recipes/radgeetoolbox/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - earthengine-api
+    - python >=3.6
+
+test:
+  imports:
+    - RadGEEToolbox
+
+about:
+  home: "https://github.com/radwinskis/RadGEEToolbox"
```
Comment (ocefpaf) [inline recipes/radgeetoolbox/meta.yaml:30]:
```suggestion
  home: https://github.com/radwinskis/RadGEEToolbox
```

**Pull request merged**